src/components/NBDialog.tsx

```diff
@@ -12,7 +12,7 @@ function NBDialog({ open, onClose }: { open: boolean; onClose: () => void }) {
   const fullScreen = useMediaQuery(theme.breakpoints.down('md'));
 
   return (
-    <Dialog fullScreen={fullScreen} open={open} onClose={() => onClose()}>
+    <Dialog fullScreen={fullScreen} open={open} onClose={onClose} data-cy="nb-dialog">
```

Comment:
this was leftover from changing the function signature for `onClose` in a previous PR, correct?

Reply:
No, more like a refactor after we found out that we can use this simpliefied syntax for the callback function.